packages/dashboard/src/components/composed/Sidebar/Middle/index.tsx

```diff
@@ -40,6 +41,11 @@ function Middle(): JSX.Element {
           </Indicator>
         }
       />
+      <NavButton
+        label="Settings"
+        to="/Settings"
+        icon={<Settings {...featherIconProps} />}
+      />
```

Comment:
No problem with this code but I'm thinking settings shouldn't be on the sidebar, along with signature requests and contract interaction and such, as it's more tangential to what dashboard is actually for. Instead of using `<NavButton />` here, do you think the dark/light mode button right now should instead be the settings button? From there you would be able to toggle analytics, dark/light mode, etc.

Reply:
Following on that, maybe settings shouldn't be its own page but a modal instead.

Reply:
What do you think about a settings button next to the dark/light mode button that opens up a modal?

Reply:
Eh it doesn't agree with me personally.
<img src="https://user-images.githubusercontent.com/41348973/213009317-dc2732a1-0f54-4008-86d4-643dec3d4d24.png" width="400px" />

Reply:
yeah, it doesn't look pretty. Any other suggestions?

Reply:
Yea as I mentioned [above](https://github.com/trufflesuite/truffle/pull/5841#discussion_r1072703074), replacing the dark/light mode button with settings button.

Keen to hear other thoughts though.